quadcopter_brain/src/quadcopter_brain/quadcopter_brain.py

```diff
@@ -45,6 +45,24 @@ def hover_in_place(self):
         self.go_to_waypoints(waypoint_data)
         print("Hover command sent")
 
+    def go_to_faux_relative_waypoint(self, dEast, dNorth, dAlt=0,
```

Comment:
name this something more descriptive. e.g. go_to_global_relative_waypoint, go_to_close_global_waypoint, go_to_relative_gps_waypoint